Gemfile

```diff
@@ -144,6 +144,9 @@ gem "unicode-display_width"
 # Keep ruby 3.0 compatibility
 gem "multi_xml", "~> 0.6.0"
 
+gem "teaspoon"
+gem "teaspoon-mocha", "~> 2.3.3"
```

Comment:
These gems shouldn't be in the production section, either `group :test` or `group :development, :test`

Since teaspoon-mocha depends on teaspoon, there's no need to add it separately

Reply:
Moved the gems to `group :development, :test` which made a bit more sense to me since running tests via command line is a bit more straight forward. Thank you for review!